terraform/031-email-service/main.tf

```diff
@@ -138,16 +137,6 @@ module "ecsservice_cron" {
  * Create Cloudflare DNS record(s)
  */
 resource "cloudflare_record" "emaildns" {
-  count = var.create_dns_record ? 1 : 0
```

Comment:
Can this `create_dns_record` variable now be removed from the 031-email-service module?

Reply:
I think the secondary region still needs to set it to `false`.

Reply:
It looks like it's not used anywhere any more (in the 031-email-service module), and now that it's only creating a region-specific domain name, I don't think it's applicable any more. Thoughts?

Reply:
I must have missed something before. I see now what you're saying.